### ISSUE TITLE
YConsolidatedDataSet.cs error for UWP

### ISSUE THREAD

**Issue by brettmilinski** [yoctopuce/yoctolib_uwp#1]
Hello, I recently downloaded your UWP examples from GitHub and I have ran into errors using your YConsolidatedDataSet.cs source code.

There are errors when trying to receive sensor measures and data with pointers. The current code for example:
1. **newdataset=sensor.get_recordedData(_start,_end)**
2. **measures=_datasets[s]->get_measures();**
3. **currnexttim=measures[idx]->get_endTimeUTC();**
4. **measures = _datasets[s]->get_measures();**
5. **newvalue = measures[idx]->get_averageValue();**
6. **currprogress =  _datasets[s]->loadMore();**

Produce an error of: "Error CS0193	The * or -> operator must be applied to a pointer"

Changing these pointer statements to the following solved the issue:
1. **newdataset=await sensor.get_recordedData(_start, _end);**
2. **measures=await _datasets[s].get_measures();**
3. **currnexttim=measures[idx].get_endTimeUTC();**
4. **measures = await _datasets[s].get_measures();**
5. **newvalue = measures[idx].get_averageValue();**
6. **currprogress = await _datasets[s].loadMore();**

However, I still have the following errors in the code below:
```
protected YConsolidatedDataSet(YAPIContext ctx, string func)
        : base(ctx, func, "ConsolidatedDataSet")
 {
        //--- (generated code: YConsolidatedDataSet attributes initialization)
        //--- (end of generated code: YConsolidatedDataSet attributes initialization)
        this._init(startTime, endTime, sensorList);
 }
```
1. **: base(ctx, func, "ConsolidatedDataSet")** 
Produces error "CS1729 'object' does not contain a constructor that takes 3 arguments"
2. **this._init(startTime, endTime, sensorList);** 
Produces 3 errors of "CS0103 variable name does not exist in the current context"
Are these variables meant to be **_start,_end,_sensors**?

Can changes be made to fix these issues?

If this is me implementing the software wrong, please correct me.

Thank you for your time,
Brett

**Comment by yoctopuce** [comment]
Oops. Clearly the file YConsolidatedDataSet.cs is wrong. I will fix that and see how this happened.

**Comment by yoctopuce** [comment]
We have push the fix (99f181b3d5a983e6ea25270bde9ff254bbe0d903) for this error. Juste update your git repository  and your code should work as expected.

Note: We will release a new official build with this fix and the missing Prog-ExportCSV example

**Comment by brettmilinski** [comment]
@yoctopuce Thank you for the quick response and fix. Have a great week!
